File: z-clients/rust/src/models/ready_out.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct ReadyOut {
    pub ok: bool,

    pub message: String,
}

impl ReadyOut {
    pub fn new(ok: bool, message: String) -> Self {
        Self { ok, message }
    }
}

File: z-clients/javascript/src/models/readyOut.ts
// this file is @generated

export interface ReadyOut {
    ok: boolean;
    message: string;
}

export const ReadyOutSerializer = {
    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _fromJsonObject(object: any): ReadyOut {
        return {
            ok: object['ok'],
            message: object['message'],
        };
    },

    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _toJsonObject(self: ReadyOut): any {
        return {
            'ok': self.ok,
            'message': self.message,
        };
    }
}